velox/functions/sparksql/aggregates/tests/BloomFilterAggAggregateTest.cpp

```diff
@@ -71,14 +71,15 @@ TEST_F(BloomFilterAggAggregateTest, emptyInput) {
   testAggregations(vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expected);
 }
 
-TEST_F(BloomFilterAggAggregateTest, nullBloomFilter) {
-  auto vectors = {makeRowVector({makeAllNullFlatVector<int64_t>(2)})};
-  auto expectedFake = {makeRowVector(
-      {makeNullableFlatVector<StringView>({std::nullopt}, VARBINARY())})};
-  VELOX_ASSERT_THROW(
-      testAggregations(
-          vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expectedFake),
-      "First argument of bloom_filter_agg cannot be null");
+TEST_F(BloomFilterAggAggregateTest, withNullInput) {
+  // The first 10 values range from 0 to 9, and the last 10 values are nulls
+  auto vectors = {makeRowVector({makeFlatVector<int64_t>(
```

Comment:
This can be written in a more reader-friendly way.

```
makeNullableFlatVector<int64_>({0, 1, 2, std::nullopt, 3, 4, 5, std::nullopt, std::nullopt, 6, 7, 8, 9, std::nullopt, std::nullopt, std::nullopt});
```